app/services/form_duplicator.rb

```diff
@@ -27,6 +27,7 @@ def new_form
       @new_form
     else
       @new_form = @form.dup
+      @new_form.name = @form.name + "-#{SecureRandom.uuid.to_s}"
```

Comment:
Saving a form with a duplicate name is now broken, so I felt like appending a UUID was the best way to make sure we didn't have collisions. Open to other options, especially if anyone wants to implement them while I'm gone!